.travis.yml

```diff
@@ -21,8 +21,7 @@ rvm:
   - ruby-head
   - jruby
   - jruby-head
-  - rbx-2
-  - rbx-3
+  - truffleruby
```

Comment:
Let's leave rbx as-is and add truffleruby in a separate PR. This PR can focus on integration testing with Action Mailer alone.

Reply:
ok